frontend/src/components/admin/Admin.js

```diff
@@ -108,19 +102,26 @@ function Admin() {
             </SideNavMenuItem>
           </SideNavMenu>
 
+          <SideNavMenu
```

Comment:
Do not refactor theose side nave menu items , 
just add   `<SideNavLink>` to the new Dictionary Menu Page

Reply:
Ooh I got you now. Lemi fix this in the next (hopefully last commit) of this PR

Reply:
dont add a 
   ```
<SideNavMenu
       renderIcon={Microscope}
            title={intl.formatMessage({ id: "master.lists.page" })}
          >
            <SideNavMenuItem href="#DictionaryMenu">
              <FormattedMessage id="dictionary.label.modify" />
            </SideNavMenuItem>
          </SideNavMenu>
```
just add a 

       ```
 <SideNavLink href="#DictionaryMenu" renderIcon={CharacterWholeNumber}>
            <FormattedMessage id="dictionary.label.modify" />
          </SideNavLink>
```